doc/DESI_SPECTRO_DATA/NIGHT/EXPID/desi-EXPID.rst

```diff
@@ -16,138 +16,302 @@ There is one HDU per spectrograph camera with EXTNAMEs like
+OFFSET0   0.4000000059604645,-8.8683          str   [V] set value, measured value
+OFFSET1   0.4000000059604645,-8.8683          str   [V] set value, measured value
+OFFSET2   0.4000000059604645,-8.8374          str   [V] set value, measured value
+OFFSET3   0.4000000059604645,-8.8786          str   [V] set value, measured value
+OFFSET4   2.0,6.0174                          str   [V] set value, measured value
+OFFSET5   2.0,6.0648                          str   [V] set value, measured value
+OFFSET6   2.0,6.0174                          str   [V] set value, measured value
+OFFSET7   2.0,6.0437                          str   [V] set value, measured value
+DELAYS    30, 30, 25, 40, 7, 3000, 7, 7, 7, 7 str   [10] Delay settings
+PGAGAIN   3                                   int   Controller gain
+BLDTIME   0.3355                              float [s] Time to build image
+DIGITIME  48.0327                             float [s] Time to digitize image
+OBSNUM    1000                                int
+FEEBOX    lbnl053                             str
+VESSEL    2                                   int
+CDSPARAMS na, na, na, na                      str
+CHECKSUM  9aEAAZB53aBA9YB5                    str   HDU checksum updated 2019-02-20T21:27:58
+DATASUM   1562144619                          str   data unit checksum updated 2019-02-20T21:27:58
+GAINA     1.0                                 float Gains from ICS **MISSING**
+GAINB     1.0                                 float **MISSING**
+GAINC     1.0                                 float **MISSING**
+GAIND     1.0                                 float **MISSING**
+RDNOISEA  3.0                                 float Expected readnoise from ICS, not measured from these data **MISSING**
+RDNOISEB  3.0                                 float **MISSING**
+RDNOISEC  3.0                                 float **MISSING**
+RDNOISED  3.0                                 float **MISSING**
+INHERIT   T                                   bool  https://fits.gsfc.nasa.gov/registry/inherit.html **MISSING**
+========= =================================== ===== ==============================================
+
+Data: FITS image [int16 (compressed), 4256x4194]
+
+HDU3 -- HDU31
+-------------
+
+EXTNAME = B1, B2, B3, B4, B5, B6, B7, B8, B9, R0, R1, R2, R3, R4, R5, R6, R7, R8, R9, Z0, Z1, Z2, Z3, Z4, Z5, Z6, Z7, Z8, Z9
+
+Data: See B0.
 
 Notes and Examples
 ==================
 
-*Add notes and examples here.  You can also create links to example files.*
+Provenance
+----------
+
+* 2019-02-21: Revised based on headers from spectrograph functional verification files.
+
+Problems
+--------
+
+* The compressed HDUs in the "sp0" files contain ``ZSIMPLE`` keyword.  This would
+  be appropriate in a compressed *primary* HDU but not in a compressed extension.
+  Make sure that the images are actually compressed *as extensions*, not as
+  individual images that are then shoved into an HDU.
```

Comment:
OK to have notes here, but we'll likely need a different way to converge with Klaus/ICS about issues like this (i.e. don't expect Klaus to be reading the datamodel and responding...)

Reply:
Noted.